File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let check = new Set()
    for(let num of nums){
        if (check.has(num)){
            return true
        }
        else{
            check.add(num)
        }

    }
     return false;
};

File: 217-contains-duplicate/contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        check = set()
        for i in nums:
            if i in check:
                return True
            else:
                check.add(i)
        return False
            
        